<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchForm</class>
 <widget class="QWidget" name="SearchForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>جستجو</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    background-color: rgb(228, 229, 244); /* Light gray background */
    border: 1px solid #a9a9a9; /* Darker gray border for a 3D effect */
    border-radius: 6px; /* Rounded corners */
    padding: 5px 10px; /* Padding for a button-like appearance */
    color: black; /* Text color */
    
    /* Subtle gradient for a shiny effect */
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, 
                                stop:0 #f0f0f0, stop:1 #d3d3d3);
}

QPushButton:hover {
    background-color: #e6e6e6; /* Change to a lighter shade on hover */
    border-color: #b0b0b0; /* Change border color on hover */
}

QPushButton:pressed {
    background-color: #bfbfbf; /* Darker gray when pressed */
    border-color: #808080; /* Slightly darker border when pressed */
}


QWidget {font: 11pt &quot;Segoe UI&quot;;}
QWidget {
    background-color: #d0dae1; /* Dark background color for the entire widget */
    font: 11pt &quot;Segoe UI&quot;;
}

/* Style for QLineEdit with a darker background and smoother edges */
QLineEdit {
    background-color: #eff3f5; /* Darker grayish background */
    color: #000000; /* Light text color for readability */
    border: 1px solid #5c5c70; /* Slightly lighter border for contrast */
    border-radius: 6px; /* Smooth, rounded edges */
    padding: 5px; /* Padding for a cleaner look */
    font: 11pt &quot;Segoe UI&quot;;
}

QLineEdit:focus {
    border-color: #7878a0; /* Lighter border color on focus */
    background-color: #e0e6eb; /* Slightly lighter background on focus */
}

QTextBrowser {
    background-color: #eff3f5; /* Darker grayish background */
    color: #000000; /* Light text color for readability */
    border: 1px solid #5c5c70; /* Slightly lighter border for contrast */
    border-radius: 6px; /* Smooth, rounded edges */
    padding: 5px; /* Padding for a cleaner look */
    font: 11pt &quot;Segoe UI&quot;;
}

QTextBrowser:focus {
    border-color: #7878a0; /* Lighter border color on focus */
    background-color: #e0e6eb; /* Slightly lighter background on focus */
}

QCheckBox {
    color: #000000; /* Light text color for dark background */
    font: 11pt &quot;Segoe UI&quot;;
}

QCheckBox:disabled {
    color: #000000; /* Same text color for disabled state */
    opacity: 1; /* Remove opacity reduction */
}

/* Blue indicator for checked and disabled QCheckBox */
QCheckBox::indicator:checked:disabled {
    background-color: #92a8b9; /* Blue color to appear as enabled */
	opacity: 1;
   /* border: 1px solid #3a7bd5;  Blue border to match the indicator */
}

QCheckBox::indicator:disabled {
    opacity: 1; /* Remove opacity change for checkbox indicator */
}

/* Background color for QWidget */
/* Style for the vertical scrollbar */
QScrollBar:vertical {
    background: #eff3f5; /* Background color of the scrollbar */
    width: 10px; /* Width of the scrollbar */
    margin: 0px 0px 0px 0px; /* Margins around the scrollbar */
    border-radius: 5px; /* Rounded edges for the scrollbar track */
}

/* Style for the horizontal scrollbar */
QScrollBar:horizontal {
    background: #eff3f5; /* Background color of the scrollbar */
    height: 10px; /* Height of the scrollbar */
    margin: 0px 0px 0px 0px; /* Margins around the scrollbar */
    border-radius: 5px; /* Rounded edges for the scrollbar track */
}

/* Handle (scroller) styling for both horizontal and vertical scrollbars */
QScrollBar::handle:vertical, QScrollBar::handle:horizontal {
    background: #92a8b9; /* Color of the scroller */
    min-height: 20px; /* Minimum height for the vertical handle */
    min-width: 20px; /* Minimum width for the horizontal handle */
    border-radius: 5px; /* Smooth, rounded edges for the handle */
}

QScrollBar::handle:vertical:hover, QScrollBar::handle:horizontal:hover {
    background: #b1c1cd; /* Slightly lighter color on hover for the handle */
}

/* Remove the top/bottom buttons on the scrollbar */
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical,
QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    background: none; /* No additional lines/buttons */
    height: 0px;
    width: 0px;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical,
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none; /* No page scroll styling */
}
QTextEdit {
    background-color: #eff3f5; /* Darker grayish background */
    color: #000000; /* Light text color for readability */
    border: 1px solid #5c5c70; /* Slightly lighter border for contrast */
    border-radius: 6px; /* Smooth, rounded edges */
    padding: 5px; /* Padding for a cleaner look */
    font: 11pt &quot;Segoe UI&quot;;
}

QTextEdit:focus {
    border-color: #7878a0; /* Lighter border color on focus */
    background-color: #e0e6eb; /* Slightly lighter background on focus */
}
QComboBox {
    background-color: #eff3f5; /* Darker grayish background */
    color: #000000; /* Light text color for readability */
    border: 1px solid #5c5c70; /* Slightly lighter border for contrast */
    border-radius: 6px; /* Smooth, rounded edges */
    padding: 5px; /* Padding for a cleaner look */
    font: 11pt &quot;Segoe UI&quot;;
}

QComboBox:focus {
    border-color: #7878a0; /* Lighter border color on focus */
    background-color: #e0e6eb; /* Slightly lighter background on focus */
}
QDateEdit {
    background-color: #eff3f5; /* Darker grayish background */
    color: #000000; /* Light text color for readability */
    border: 1px solid #5c5c70; /* Slightly lighter border for contrast */
    border-radius: 6px; /* Smooth, rounded edges */
    padding: 5px; /* Padding for a cleaner look */
    font: 11pt &quot;Segoe UI&quot;;
}

QDateEdit:focus {
    border-color: #7878a0; /* Lighter border color on focus */
    background-color: #e0e6eb; /* Slightly lighter background on focus */
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>جستجو در :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_4"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>متن جستجو :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="comboBox_2"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>نوع قطعه :</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QComboBox" name="comboBox_3"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>نام قطعه :</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>lineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
